pkg/lib/machine-change-auth.html

```diff
@@ -1,93 +1,84 @@
+    and the host does not allow logging in with a password. Please
+    provide the password of the key at {{#strong}}{{key}}{{/strong}}.
+    </span><span translate="yes" class="password-change-advice">You
+    may want to change the password of the key for automatic
+    login.</span></p>
+    {{/only_key}}
 
-            {{#password}}
-            <tr id="login-diff-password">
-                <td translate="yes" class="top">Password</td>
-                <td>
-                    <input class="form-control" id="login-custom-password" type="password" />
-                </td>
-                <td>
-                    <a tabindex="0" role="button" data-toggle="popover"
-                        data-trigger="focus" data-placement="bottom" translate="data-content"
-                        data-content="Entering a different password here means you will need to retype it every time you reconnect to this machine">
-                        <span class="fa fa-lg fa-info-circle"></span>
-                    </a>
-                </td>
-            </tr>
-            {{/password}}
-            <tr id="login-available">
-                <td translate="yes" class="top">Available</td>
-                <td class="top">
-                        {{#show_password}}<div translate>Login Password</div>{{/show_password}}
-                        {{#show_ticket}}<div translate>Kerberos Ticket</div>{{/show_ticket}}
-                        {{^show_password}}{{^show_ticket}}<div class="empty hidden">No available credentials</div>{{/show_ticket}}{{/show_password}}
-                        {{#public-key}}<div class="keys"><div class='dialog-wait-ct pull-left'><div class='spinner spinner-sm'></div><span translate="yes">Checking for public keys</span></div></div>{{/public-key}}
-                </td>
-            </tr>
-            {{#show_locked_identity}}
-            <tr class="login-locked">
-                <td translate="yes" class="top">Locked</td>
-                <td class="top key-locked form-group">
-                    <div class="locked-identity"></div>
-                    <div translate="yes">Enter passphrase to add identity to ssh-agent</div>
-                    <input class="locked-identity-password" type="password" autocomplete="new-password"></input>
-                    <button class="pf-c-button pf-m-primary" translate="yes">Unlock</button>
-                </td>
-            </tr>
-            {{/show_locked_identity}}
-        </table>
-    </div>
+    {{#password_and_key}}
+    <p><span translate="yes">The SSH key for logging
+    in to {{#strong}}{{full_address}}{{/strong}} is protected. You can
+    log in with either your login password or by providing the
+    password of the key at
+    {{#strong}}{{key}}{{/strong}}.</span><span translate="yes"
+    class="password-change-advice">You may want to change the password
+    of the key for automatic login.</span></p>
+    {{/password_and_key}}
+
+    <br/>
+
+    <form class="ct-form">
```

Comment:
Html spacing in cockpit is 4 spaces

Reply:
I thought `npm run eslint:fix` takes care of HTML as well, no?

Reply:
No, eslint only fixes *.js{,x} files.